src/main.rs

```diff
@@ -157,7 +165,7 @@ async fn handler(
                             let mut player_data = data.player_data.lock().unwrap();
                             player_data
                                 .entry(new.user_id)
-                                .or_insert(config.default_player_data.clone()).queue_enter_time = None;
+                                .or_insert_with(PlayerData::default).queue_enter_time = None;
```

Comment:
It's actually intentional to not use default here: I want server staff to be able to modify default player data.

Reply:
ah makes sense